notebooks/Brevitas_TVMCon2021.ipynb

```diff
@@ -2252,21 +2252,20 @@
     "            for i, (images, _) in enumerate(calibration_loader):\n",
     "                print(f'Calibration iteration {i}')\n",
     "                quant_model(images)\n",
-    "        # Put the model in inference mode to use those statistics\n",
-    "        quant_model.eval()\n",
-    "        bc = BiasCorrection(iterations=len(calibration_loader))\n",
-    "        for i, (images, _) in enumerate(calibration_loader):\n",
-    "            print(f'Bias correction iteration {i}')\n",
-    "            # Apply bias correction\n",
-    "            quant_model = bc.apply(quant_model, images)\n",
+    "        \n",
+    "        # Apply bias correction\n",
+    "        with bias_correction_mode(quant_model):\n",
+    "            for i, (images, _) in enumerate(calibration_loader):\n",
+    "                print(f'Bias correction iteration {i}')\n",
+    "                quant_model(images)\n",
     "    quant_model.apply(finalize_collect_stats)\n",
     "    return quant_model"
    ]
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3.7.15 ('torch_1.10')",
+   "display_name": "Python 3.6.9 64-bit",
```

Comment:
The beginning of the tutorial should be updated to say that the tutorial was originally released for brevitas 0.7.1, but has since been updated to brevitas. 0.8.